README.md

```diff
@@ -11,15 +11,20 @@ If creating a new experiment, see [the docs](docs/new_experiments.md).
 
 ## Experiments
 
```

Comment:
Better see here than in the diff: https://github.com/faasm/experiment-base/blob/invoke/README.md